### ISSUE TITLE
[BUG] Voice Control stumbling blocks

### ISSUE THREAD

**Issue by mxmeinhold** [mastodon/mastodon-ios#644]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

There are a few actions that are somewhat difficult when using VoiceOver. This mastodon thread has screenshots and examples. https://towns.gay/@max_in_somer/109369238418377235 

### Environment

```markdown
- Device: iPhone 12 Pro
- OS: 16.2
- Version: V1.4.5
- Build: 144
```

**Comment by j-f1** [comment]
Thanks for spotting those! Responded in thread but also feel free to respond here if you prefer:

Thanks for reporting! I’ve recently contributed a bunch of accessibility improvements to the iOS app! 
1. I think that’s standard iOS behavior where if there isn’t enough space to display the full title of the previous screen (like “notifications”) it will just show “Back” visually but maybe accessibility tools will always see the full title? Do you see this happening in other apps too? 
2. Can you try this again with v1.4.7 (on the App Store now)? I think we fixed this issue (but if it is still happening let me know!)

I’m also very interested in other Voice Control issues you’re experiencing. I’ve mostly tested with just VoiceOver so there are probably missing things!
